src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -17,6 +17,7 @@
 package com.pinterest.secor.common;
 
 import com.google.api.client.repackaged.com.google.common.base.Strings;
+import com.pinterest.secor.uploader.Uploader;
```

Comment:
@HenryCaiHaiying thanks for the feedback, makes sense, will include in my next amended commit.